docs/pages/push-notifications/sending-notifications-custom.mdx

```diff
@@ -1,37 +1,37 @@
+Sometimes you need finer-grained control over your notifications, so communicating directly with FCM and APNs is necessary. The Expo platform does not lock you into using Expo's application services, and the `expo-notifications` API is push-service agnostic.
 
-## How do I write my own APNs & FCM servers?
+## How do I write my own FCM & APNs servers
 
-Before we begin communicating directly with APNs & FCM, there is one client-side change you'll need to make in your app. When using Expo's notification service, you collect the `ExponentPushToken` with [`getExpoPushTokenAsync`](../versions/latest/sdk/notifications.mdx#getexpopushtokenasyncoptions-expotokenoptions-expopushtoken). Now that you're not using Expo's notification service, you'll need to collect the native device token instead with [`getDevicePushTokenAsync`](../versions/latest/sdk/notifications.mdx#getdevicepushtokenasync-devicepushtoken).
+Before communicating directly with FCM & APNs, there is one client-side change you'll need to make in your app. When using Expo's notification service, you collect the `ExponentPushToken` with [`getExpoPushTokenAsync`](/versions/latest/sdk/notifications/#getexpopushtokenasyncoptions-expotokenoptions-expopushtoken). Now that you're not using Expo's notification service, you'll need to collect the native device token instead with [`getDevicePushTokenAsync`](/versions/latest/sdk/notifications/#getdevicepushtokenasync-devicepushtoken).
 
 ```diff
 import * as Notifications from 'expo-notifications';
 ...
 - const token = (await Notifications.getExpoPushTokenAsync()).data;
 + const token = (await Notifications.getDevicePushTokenAsync()).data;
-// send token off to your server
+// send token to your server
 ```
 
-Now that you have your native device token, we can start to implement our servers. Below are some very minimal examples of communicating with FCM and APNs:
+After getting the native device token, you can start implementing the servers. Below are some minimal examples of communicating with FCM and APNs:
 
-## FCM Server
+## FCM server
 
-> This documentation is based off of [Google's documentation](https://firebase.google.com/docs/cloud-messaging/http-server-ref), and we're just going to cover the basics here to get you started.
+> This documentation is based on [Google's documentation](https://firebase.google.com/docs/cloud-messaging/http-server-ref), and this section covers the basics to get you started.
 
-Communicating with FCM is as simple as sending a POST request, but before sending or receiving any notifications, you'll need to follow the steps [in this documentation](using-fcm.mdx) to configure FCM (and get your `FCM-SERVER-KEY`).
+Communicating with FCM is as simple as sending a POST request. However, before sending or receiving any notifications, you'll need to follow the steps to [configure FCM](/push-notifications/push-notifications-setup/#android) to configure FCM and get your `FCM-SERVER-KEY`.
```

Comment:
```suggestion
Communicating with FCM is done by sending a POST request. However, before sending or receiving any notifications, you'll need to follow the steps to [configure FCM](/push-notifications/push-notifications-setup/#android) to configure FCM and get your `FCM-SERVER-KEY`.
```
We say "is as simple as a POST request" and then immediately explain that it's harder than that 😂 --Just taking out that statement here.